### ISSUE TITLE
No sound in menus/game play in Call Of Duty : Black Ops

### ISSUE THREAD

**Issue by Eignar17** [wine-compholio/wine-staging#146]
I think several games are impacted by this bug in sound.

https://bugs.winehq.org/show_bug.cgi?id=25173

**Comment by deri82** [comment]
I just downloaded today the full version and tried it with crossover version of wine. I could hear sounds. But the game is missing textures, most likely DX_something compressed. 

Well then i tried pure wine, just vcrun6 package preinstalled because testing other bug. 

I couldnt get this run at all. 

fixme:d3d:debug_d3dformat Unrecognized 0x41415353 (as fourcc: SSAA) WINED3DFORMAT!
fixme:d3d:wined3d_get_format Can't find format unrecognized (0x41415353) in the format lookup table

**Comment by slackner** [comment]
@deri82: well, you should probably try running with wine staging. It is always nice to know which bugs are already fixed by our patchsets. Also, since your error description does not match at all the initial bug report (missing sound) it means that there is either something wrong with your setup, or that the bug is currently blocked by regression. If you are sure that you followed all hints on appdb / tested with a clean prefix / ... and so on, and it still doesn't work, it probably makes sense to open a new upstream bug report for this new issue.

**Comment by slackner** [comment]
Hi everyone,

About one year ago, we announced that we would like to move to more advanced bug tracker. We are sorry for the inconvenience, but now its time to say finally goodbye to the old one. This means, all remaining opened bugs (including this one!) will be closed, and no longer available. If you still need any information from the old bugtracker, please write a mail to webmaster@fds-team.de.

More important, if your issue still exists, please open a new bug report at one of the places mentioned below. For bug reports, please attach terminal output from a recent Wine version (preferably >= 1.8). Please note that in the meantime, Wine Staging has become an official part of WineHQ, so if your issue is already tracked at winehq.org, there is NO NEED to open a new bug report for it. Feel free to add the Wine Staging maintainers (sebastian@fds-team.de, michael@fds-team.de) as CC if you want us to take a look at your bug report.

**NEW BUG TRACKERS:**

* For all Wine AND Wine Staging bugs. USE THIS IF IN DOUBT. (*)
  https://bugs.winehq.org/enter_bug.cgi?product=Wine

* For packaging issues:
  https://bugs.winehq.org/enter_bug.cgi?product=Packaging

* For patch submission:
  https://bugs.wine-staging.com/enter_bug.cgi

Best regards,
Sebastian

----

(*) The "Wine Staging" product in the winehq bugtracker should only be used, when your problem turns out to be a regression introduced in "Wine Staging". Since most problems also occur with Wine, it is recommended to open your bug in the "Wine" product, unless someone can show that it doesn't occur with the development branch.
